### ISSUE TITLE
What is the Color of the Smooth Stellar Component 

### ISSUE THREAD

**Issue by robelgeda** [robelgeda/dsii_dwarfs#9]
These are the low mass stars that we will not be simulating. We will need to sum up the flux in each band in the isochrone for masses below our cutoff M_indv.

**Comment by hcferguson** [comment]
Added this calculation to `HSC_dwarf_simulation.ipynb`. Having previously converted magnitudes to linear fluxes at 10pc in the isochrone, it does a numerical integral of the the flux weighted by the dN/dm from the mass function. 

```
m = isochrone['initial_mass']
dn_dm = kroupa(m)
smooth_flux = {}
selection = isochrone['initial_mass'] <= minmass_stochastic
for b in bands:
    smooth_flux[b] = integrate.simps(m[selection],isochrone[b+'flux'][selection]*dn_dm[selection])
smooth_flux
```

**Comment by hcferguson** [comment]
Closed by #17
